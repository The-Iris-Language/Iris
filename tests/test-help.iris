class Animal of Organism() {
  permit:
    int age;
    string Animal() {
      return "g";
      Cat c = new Cat(hi, 7, "bruh");
      Cat c = new Cat();
    }
    Gecko bark() {
      return "woof";
      Animal dog;
    }
}

class Main () {
  public:
    int main() {
      Animal.bark();
      Animal dog = d;
      return 0;
    }
}