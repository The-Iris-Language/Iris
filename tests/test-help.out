class Animal of Organism () {
permit:
int age;
string Animal() {
return "g";
Cat c = Cat.Cat(hi, 7, "bruh");
Cat c = Cat.Cat();
}
Gecko bark() {
return "woof";
Animal dog;
}
}

class Main of Object () {
public:
int main() {
Animal.bark();
Animal dog = d;
return 0;
}
}
